src/cli.js

```diff
@@ -92,7 +94,7 @@ function saveProblemsReport(problems) {
     }
   }
   if (problemCount > 0) {
-    const reportFileName = `md-conversion-problems-report-${now.toISOString()}.md`;
+    const reportFileName = `md-conversion-problems-report-${now.toISOString().replace(/:/g, "_")}.md`;
```

Comment:
windows file path is not support `:`

https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions